### ISSUE TITLE
Email drag-drop with SuiteCRM email client is very slow

### ISSUE THREAD

**Issue by domtylerlovett** [salesagility/SuiteCRM#2691]
#### Issue
Dragging/dropping one or more emails to other email folders is very slow & synchronous.

#### Expected Behavior
It should be a much faster user experience, preferably running asynchronously.

#### Actual Behavior
Drag/drop one or more emails is very slow.

#### Possible Fix
Make it run asynch.

#### Steps to Reproduce
1. Drag/drop an email from the Inbox to another email folder.

#### Context
I have multiple clients using SuiteCRM and they all love everything except the emails!
I think this should be high priority as it makes the email client largely unusable when other email clients are available that are far better.

#### Your Environment

* SuiteCRM Version used: 7.7.8
* Browser name and version: Chrome (Latest)
* Environment name and version: MySQL, PHP
* Operating System and version: Windows 10

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Email drag-drop with SuiteCRM email client is very slow](https://trello.com/c/Yr28Q5ye/50-email-drag-drop-with-suitecrm-email-client-is-very-slow)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
